html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <label for="email">Email
+      <br>
+      <input type="text" name="email" id="email">
+    </label>
+    <div>
+      <label for="reason">Why is coding cool?
+        <br>
```

Comment:
make sure to remove this `br` tag and any other `br` tags. Those are only for text formatting, not layout.